pkgs/tools/security/nitrokey-app2/default.nix

```diff
@@ -17,11 +19,19 @@ python3.pkgs.buildPythonApplication rec {
     hash = "sha256-VyhIFNXxH/FohgjhBeZXoQYppP7PEz+ei0qzsWz1xhk=";
   };
 
+  # https://github.com/Nitrokey/nitrokey-app2/issues/152
+  postPatch = ''
+    substituteInPlace pyproject.toml --replace "pynitrokey ==" "pynitrokey >="
+  '';
```

Comment:
```suggestion
  pythonRelaxDeps = [
    "pynitrokey"
  ];
```

Reply:
I tried this, but it didn't work.

Reply:
Ah I see, I tried it myself now. The [`pythonRelaxDepsHook`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/interpreters/python/hooks/python-relax-deps-hook.sh) is executed in the `postBuild` phase and only modifies the dependencies in the built distribution. I'm not sure if there is a better way than this substitution but it works fine.